src/utils.rs

```diff
@@ -1,6 +1,7 @@
 use gelatin::glium::glutin::event::VirtualKeyCode;
 
 pub fn virtual_keycode_is_char(vk: VirtualKeyCode) -> bool {
+	#[allow(clippy::match_like_matches_macro)]
```

Comment:
You absolutely made the right call here, in my opinion. This would look ugly with a `matches!` macro.